### ISSUE TITLE
Lite terminal - default workspace folder

### ISSUE THREAD

**Issue by Patresko** [intersystems-community/vscode-objectscript#1461]
Hello,

Is there any option how to set default workspace folder to do not pick every time when I want to open lite terminal in VS code?

![image](https://github.com/user-attachments/assets/46b3297a-6f43-45d2-8bbc-ef53b4c87801)

**Comment by isc-bsaviano** [comment]
Hi @Patresko, in the past it would use the server connection for the active text editor by default, but this was removed because it was confusing some users. I see the value in saving the click, but I also think that requiring the click helps avoid confusion. I'll leave this open to see if anyone else wants to weigh in.

**Comment by Patresko** [comment]
I guess would be nice if there will be option for whose want to use default server connection in active editor and whose does not, they can select workspace folder. 

But I see when I open via command pallet, then is open directly without choosing from where to get server connection. But from terminal tab, I need to prompt.

![image](https://github.com/user-attachments/assets/1037e351-5c49-4ce7-912b-ad51507b7e06)

**Comment by gjsjohnmurray** [comment]
@isc-bsaviano do we dedup the entries and only make the user choose when their namespace connects to more than one server:namespace? And in that case maybe the choice would be better presented in terms of server:namespace than root folder name?

**Comment by Patresko** [comment]
@isc-bsaviano  Is not possible to just add option for the plugin if you want to use same connection information for both of terminals ? By the default can be turn off, but users like me can always check to override default behaviour. 

Like I have mapped 3 different servers in 3 different workspace, so for me its not confusing on which server currently Im working and clicking to confirm connection for terminal is quiet annoying for me.

![image](https://github.com/user-attachments/assets/d2dce5a3-56fe-49cd-8164-176e3fb5d838)

**Comment by isc-bsaviano** [comment]
@Patresko If I added a configuration setting every time someone disagreed with a UX decision we'd have a hundred of them. I don't think this single extra click is that bad of a price to pay for clarity. This is in line with how other terminals operate in multi-root workspace. For example, if I try to open a new `zsh` shell on my Mac from that terminal menu, it will ask me to pick a workspace folder to use as the current working directory. This is also still faster than typing `iris terminal <instance>` from a standard OS terminal.

**Comment by Patresko** [comment]
Yes I understood if this is for multi-root workspace, but if you have single workspace within one server its annoying to confirm all the time. Like you said before it was default option at the begening and for some confused users was changed. 

So maybe stupid question, but there were more confused user than user whose were happy how does it worked before? 

In my mac does not asking me even If im trying to open zsh. Because I do not have multi root workspace. I click zsh and terminal is directly loaded.

**Comment by isc-bsaviano** [comment]
@Patresko Can you send me your workspace definition please? If you only have one workspace folder then the extension shouldn't be asking you to choose.

**Comment by Patresko** [comment]
Sorry, I said it wrong, I have multiple folders but within same server.

`"folders": [
		{
			"name": "Classes",
			"uri": "isfs://intranet-be:intranet/?filter=*.cls"
		},
		{
			"name": "Routines",
			"uri": "isfs://intranet-be:intranet/?filter=*.mac"
		},
		{
			"name": "CSP Files",
			"uri": "isfs://intranet-be:intranet/?csp"
		}
	],`

**Comment by isc-bsaviano** [comment]
Thanks for clarifying that. I agree with @gjsjohnmurray that it would be a good enhancement to change the picker from "workspace folders" to "current server connections". Since you only have one, you wouldn't be prompted. I will take this on as an enhancement.

**Comment by isc-bsaviano** [comment]
I will tackle this after #1401 is merged.
